backend/src/repository/crud/chat.py

```diff
@@ -74,11 +67,13 @@ async def read_chat_history_by_id(self, id: int) -> ChatHistory:
 
         return query.scalar()  # type: ignore
 
-    async def read_chat_history_by_session_id(self, id: int) -> typing.Sequence[ChatHistory]:
+    async def read_chat_history_by_session_id(self, id: int, limit_num=50) -> typing.Sequence[ChatHistory]:
```

Comment:
As we discussed before, if we can do  summary for current response. We can also do summary for the chat history. How do you think?

If we just do the limit without any of the semantic operations, we will lost the mean of saving the history conversation pairs.

Reply:
i leave this function as before only by adding the limit number.  Because it's not only used by load conversation , it's also used by the api of fetching chat history . It can be used to show the previous chats in the session window.  It still neccessary for us to have the raw chat saved

Reply:
Let's think about how many chat history we can keep in one session.